pkg/cmd/init.go

```diff
@@ -139,17 +139,29 @@ func initCerts(cfg *config.MicroshiftConfig) ([]byte, *cryptomaterial.Certificat
 			"kubelet-signer",
 			cryptomaterial.KubeletCSRSignerSignerCertDir(certsDir),
 			cryptomaterial.KubeControllerManagerCSRSignerSignerCAValidityDays,
-		).WithClientCertificates(
```

Comment:
So instead of creating the client certificates for kubelet using the kubelet signer, you're creating a new signer using a sub CA (kube-csr-signer) that provides client AND serving certificates for kubelet. Is this correct?

Reply:
Is this aligned with OpenShift's cert management? https://github.com/openshift/api/tree/master/tls/docs